modules/openthread/platform/radio.c

```diff
@@ -631,9 +631,8 @@ otError otPlatRadioSleep(otInstance *aInstance)
 	    sState == OT_RADIO_STATE_RECEIVE ||
 	    sState == OT_RADIO_STATE_TRANSMIT) {
 		error = OT_ERROR_NONE;
-		if (radio_api->stop(radio_dev)) {
-			sState = OT_RADIO_STATE_SLEEP;
-		}
+		radio_api->stop(radio_dev);
+		sState = OT_RADIO_STATE_SLEEP;
```

Comment:
Why this change? It's not listed in the commit message.

Reply:
Forgot to add it to the message. This fixes wrong check of `radio_api->stop()` return value
in `otPlatRadioSleep()` (api returns 0 on success, so sState was never set to `OT_RADIO_STATE_SLEEP`). 
Updated commit message.